includes/subscriptions/templates/html-wcpay-deactivate-warning.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Admin WC Subscriptions plugin warning template.
+ *
+ * @package WooCommerce\Payments
+ */
+
+?>
+<script type="text/template" id="tmpl-wcpay-plugin-deactivate-warning">
+	<div id="wcpay-plugin-deactivate-warning-notice" class="wc-backbone-modal">
+		<div class="wc-backbone-modal-content">
+			<section class="wc-backbone-modal-main" role="main">
+				<header class="wc-backbone-modal-header">
+					<h1><?php esc_html_e( 'Are you sure?', 'woocommerce-payments' ); ?></h1>
+					<button class="modal-close modal-close-link dashicons dashicons-no-alt">
+						<span class="screen-reader-text">Close modal panel</span>
+					</button>
+				</header>
+				<article>
+					<p>
+						<?php
+							printf(
+								// Translators: placeholders are opening and closing strong HTML tags.
+								esc_html__( 'Your store (might) have active WCPay Subscriptions. If you deactivate the %1$sWooCommerce Payments%2$s plugin these subscriptions will continue renewing.', 'woocommerce-payments' ),
+								'<strong>',
+								'</strong>'
+							);
+							?>
+						</br>
+						<?php
+							printf(
+								// Translators: $1 and $2 placeholders are opening and closing strong HTML tags. $3 and $4 are opening and closing link HTML tags. $5 is an opening link HTML tag.
+								esc_html__( 'PLACEHOLDER Existing subscriptions will %1$s%3$srenew manually%4$s%2$s, meaning that subscribers will need to log in to pay for renewal. Access to the advanced features of the Subscriptions extension will be removed. %5$sLearn more.%4$s', 'woocommerce-payments' ),
+								'<strong>',
+								'</strong>',
+								'<a href="https://woocommerce.com/document/subscriptions/renewal-process/#section-4">',
+								'</a>',
+								'<a href="https://woocommerce.com/document/subscriptions/deactivation/">'
+							);
+							?>
+					</p>
+					<strong><?php esc_html_e( 'Are you sure you want to deactivate WooCommerce Payments?', 'woocommerce-payments' ); ?></strong>
+				</article>
+				<footer>
+					<div class="inner">
+						<button id="wcpay-plugin-deactivate-modal-submit" class="button button-primary button-large"><?php esc_html_e( 'Yes, deactivate WooCommerce Payments', 'woocommerce-payments' ); ?></button>
+						<button class="modal-close button button-secondary button-large"><?php esc_html_e( 'Cancel', 'woocommerce-payments' ); ?></button>
+					</div>
+				</footer>
+			</section>
+		</div>
+	</div>
+	<div class="wc-backbone-modal-backdrop modal-close"></div>
+</script>
```

Comment:
This template is a copy of the other warning modal. I thought that was simpler / more robust than parameterising so we have a single modal template.